src/plugins/dam/actions/batchEditPatchAssets.js

```diff
@@ -0,0 +1,21 @@
+import NodeRef from '@gdbots/schemas/gdbots/ncr/NodeRef';
+import { actionTypes } from '@triniti/cms/plugins/dam/constants';
+
+export default ({ fields, values }, files, config) => {
+  const { schemas } = config;
```

Comment:
@rsumilang can `schemas.patchAssets` ever be something other than `PatchAssetsV1Mixin.findOne()`?. if not, might make sense to just import that directly instead of passing a config to this action

Reply:
I think it's always going to be `PatchAssetsV1Mixin.findOne()`. If I import that directly there then the action looks a little funny because it's just passing `config` straight through without getting anything from it. That of course is totally fine but if I already have what I need in the config then I think it makes sense to get it from there instead of importing it running `findOne()` again to get the value of something I already have mainly because I can't remove the `config` object, the saga is going to need additional things that the schemas object provides.

Reply:
@rsumilang up to you i guess...not sure why you have to pass in the config though. couldnt you just do something like:

```
import NodeRef from '@gdbots/schemas/gdbots/ncr/NodeRef';
import { actionTypes } from '@triniti/cms/plugins/dam/constants';
import AssetPatchedV1Mixin from '@triniti/schemas/triniti/dam/mixin/asset-patched/AssetPatchedV1Mixin';
import PatchAssetsV1Mixin from '@triniti/schemas/triniti/dam/mixin/patch-assets/PatchAssetsV1Mixin';

export default ({ fields, values }, files) => {
  const nodeRefs = files.map((nodeRefStr) => NodeRef.fromString(nodeRefStr));

  const data = {
    paths: fields,
    ...values,
  };

  const pbj = PatchAssetsV1Mixin.findOne().createMessage(data);
  pbj.addToSet('node_refs', nodeRefs);

  const config = {
    schemas: {
      assetPatched: AssetPatchedV1Mixin.findOne(),
    },
  };

  return {
    type: actionTypes.PATCH_ASSETS_REQUESTED,
    config,
    pbj,
  };
};
```

and not include it in either the uploader or batch-edit schemas? the way it is now it is being done twice. since this action only ever does one thing it could go here, unlike other actions which might do update-artice and update-video and therefore need a config

Reply:
If I do this here then I will have to change this in the delegate to import PatchAssetsV1Mixin as well for the creating form function. At that point it starts to feel like leaving the schemas file alone is fine because more than one thing is referencing it and to change it is only in one familiar location (schemas.js).